packages/uniforms-bootstrap3/src/components/fields/SelectField.js

```diff
@@ -64,4 +64,3 @@ const Select = props =>
 ;
 
 export default connectField(Select);
-
```

Comment:
not sure why the commit removed the line, its showing in my editor, but its not committing it back with the newline at the end, strange.